py3-jupyterhub.yaml

```diff
@@ -35,7 +35,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - nodejs-16
+      - nodejs-18
```

Comment:
Note this is basically a build-time only dependency of npm. Because we only call "npm" and it needs "a nodejs" but doesn't care which one. So I think this can be actually updated to just generic `nodejs`.